manifests/cf/diego.yml

```diff
@@ -13,6 +13,11 @@ instance_groups:
     release: diego
     properties:
       diego:
+        loggregator:
+          use_v2_api: true
+          ca_cert: (( grab meta.certs.loggregator.ca ))
+          cert: (( grab meta.certs.loggregator.metron.server.cert ))
+          key: (( grab meta.certs.loggregator.metron.server.key ))
```

Comment:
I've seen this block a bunch; does it make sense to meta it and just `(( grab meta.loggregator.config ))`?

Reply:
I'll have this sorted out for #24